src/main/java/uk/gov/hmcts/reform/pdf/service/client/PDFServiceClient.java

```diff
@@ -24,19 +24,19 @@
 
 public class PDFServiceClient {
 
-    private static final String GENERATE_FROM_HTML_ENDPOINT_PATH = "/html";
     private static final Logger LOGGER = LoggerFactory.getLogger(PDFServiceClient.class);
-
-    private final RestTemplate restTemplate;
+    private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
```

Comment:
General convention is that all uppercase names are used with simple type constants. For complex types it's camel case, i.e. `objectMapper`.

Reply:
I've followed existing practice used for, i.e. `LOGGER`.

Reply:
I think this should use dependency injection, teams configure their objectmapper a lot and they should expect it to be used...

Reply:
As for the convention, I missed the fact that it's already like this in the project.

Reply:
@timja-kainos I think this client lib should know the best how to talk to pdf service api. App using this lib should not worry about http, json etc imho.